djqscsv/djqscsv.py

```diff
@@ -172,15 +171,6 @@ def _validate_and_clean_filename(filename):
     return filename
 
 
-def _safe_utf8_stringify(value):
-    if isinstance(value, six.string_types):
-        return value
-    elif isinstance(value, six.binary_type):
-        return value.encode('utf-8')
-    else:
-        return six.text_type(value).encode('utf-8')
-
-
 def _sanitize_unicode_record(field_serializer_map, record):
```

Comment:
this should maybe be `_sanitize_record` or `_textify_record` now, it no longer does anything about unicode.